docs/management/api/_exceptions-api-reusable-content.asciidoc

```diff
@@ -0,0 +1,180 @@
+// --------------------------------------------------------------------------------------------------
+//  ABOUT:
+//  This file contains a set of re-usable content that can be pulled into documentation pages that describe
+//  the API usage for endpoint artifacts (ex. Trusted Apps, event filters, etc...).
+//
+//  In order to use this content, the following must be defined in source document that "include"'s this
+//  content:
+//
+//  ATTRIBUTES:
+//  The following attributes are used through out the content to customize the information to the
+//  specific artifact type:
+//
+//  - :endpoint-artifact-list-id:
+//      The 'list_id' for the artifact type
+//      Example: endpoint_trusted_apps
+//  - attribute - :endpoint-artifact-name:
+//      The name of the artifact type (usually plural)
+//      Example: Trusted Applications
+//  - attribute: :endpoint-artifact-name-single:
+//      The name of the artifact in singular form
+//      Example: Trusted Application
+//  - attribute: :endpoint-artifact-name-sentence-start:
+//      The name of the artifact type (usually plural) with first letter capitalized
+//      Example: Trusted applications
+//  - attribute: :endpoint-artifact-name-single-sentence-start:
+//      The name of the artifact in singular form with first letter capitalized
+//      Example: Trusted application
+//
+//  DOCUMENT IDS:
+//  The reusable content in this file references the following ids which must exist in the document:
+//
+//  - [#create-item-api]
+//  - [#update-item-api]
+//  - [#exception-item-entry-schema]
+//
+// --------------------------------------------------------------------------------------------------
+
+
+// tag::endpoint-exceptions-request-payload-requirements[]
+- The `list_id` value must be `pass:a[{endpoint-artifact-list-id}]`.
+- The `namespace_type` value must be `agnostic`.
+- The `type` value must be `endpoint`.
+// end::endpoint-exceptions-request-payload-requirements[]
```

Comment:
The `type` should be `simple` for create and update. I've added another tag in my [PR](https://github.com/elastic/security-docs/pull/1643/files#diff-d692e1c82fad400f2605747303488a205b1a2f01aee33df1f25f4125f09cb394R45-R49).

Reply:
Good catch. I will change this to `simple`. I created this when documenting the `create trusted apps container`, which is only done once, so I will just remove including this there. My intent with this reusable block was to use it for create and update api (or any other API where this should be set.